gatsby-config.js

```diff
@@ -5,9 +5,10 @@ module.exports = {
       name: `Open Infrastructure team of North Rhine-Westphalian Library Service Centre (hbz)`
     },
     description: `A blog for the ETL toolkit Metafacture.`,
-    siteUrl: `http://www.metafacture.org`,
+    siteUrl: `http://www.blog.metafacture.org`,
```

Comment:
I am not sure whether we agreed on this URL. If we use it, we should soon obtain an SSL certificate for the subdomain.

Reply:
I know we not finally agreed that we use this URL but we also don't use metafacture.org as URL for the blog. Because I can't leave `siteURL` empty, I thought it would be best to fill in a URL which is probably.